CODEOWNERS

```
@@ -1,9 +1,7 @@
-/plugin/ @mattermost/toolkit
```

Maybe makes sense to use @mattermost/integrations for this? cc @hanzei

```suggestion
/plugin/ @mattermost/integrations
```

---

I'm all for @mattermost/integrations taking ownership here! Feel free to commit the suggestion if your team is onboard with that :)